src/applications/check-in/travel-claim/pages/travel-intro/index.jsx

```diff
@@ -52,17 +52,11 @@ const TravelIntro = props => {
             </p>
             <ExternalLink
               href="https://www.va.gov/health-care/get-reimbursed-for-travel-pay/"
-              target="_blank"
-              rel="noreferrer"
               eventId="travel-reimbursement-eligibility--link-clicked"
               hrefLang="en"
               eventPrefix="nav"
             >
               {t('travel-reimbursement-eligibility')}
-              <i
-                aria-hidden="true"
-                className="fas fa-external-link-alt vads-u-margin-left--1"
-              />
             </ExternalLink>
           </va-process-list-item>
           <va-process-list-item>
```

Comment:
I just wanted to give you a head's up to check how this is rendering with `uswds` set. There is a slightly different implementation where the list item header should be passed in as a prop:

https://design.va.gov/storybook/?path=/docs/uswds-va-process-list--default

```
<va-process-list uswds>
  <va-process-list-item header="Check to be sure you can request a Board Appeal">
   ....
  </va-processllist-item>
</va-process-list>
```

Reply:
Thanks for the heads up @jamigibbs! in this instance I am doing this on purpose. The heading prop works great but ended up too rigid for our design. I need an h2 here semantically but our design has it styled like and h3. It would be great if heading level and heading size were decoupled in the component. But they aren't and this is why I'm not using it here.

Reply:
@brianseek  Oh interesting! That's great feedback as the design system iterate on this component. In other components we have the option to set the header size but we add that prop as it's needed (not assuming it's needed) but it looks like we need to do that here as well! We do definitely want to keep the styling of it encapsulated within the component though so that it pulls styles from the USWDS module (instead of Formation).

I'll write up a DS ticket for this and link back to it here for y'all to keep tabs on it.

Reply:
Design system issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2593